tests/test_convert_photometry.py

```diff
@@ -386,81 +495,192 @@ def test_add_photometry_metadata(dummy_logger):
+    assert gach4h_right.injection_location == "NAcc"
+    assert gach4h_right.label == "AAV-hSyn-ACh3.8"
+    assert gach4h_right.manufacturer == "BrainVTA"
+    assert gach4h_right.description.startswith("AAV virus expressing the acetylcholine sensor GRAB-ACh3.8")
+
+    assert "rDA3m (rAAV) (left NAcc)" in nwbfile.devices
+    rda3m_left = nwbfile.devices["rDA3m (rAAV) (left NAcc)"]
+    assert isinstance(rda3m_left, Indicator)
+    assert rda3m_left.injection_coordinates_in_mm == (1.7, -1.7, -6.2)
+    assert rda3m_left.injection_location == "NAcc"
+    assert rda3m_left.label == "rAAV-hsyn-rDA3m"
+    assert rda3m_left.manufacturer == "BrainVTA"
+    assert rda3m_left.description.startswith("Recombinant AAV expressing the red-shifted dopamine sensor GRAB rDA3m")
+
+    assert "rDA3m (rAAV) (right NAcc)" in nwbfile.devices
+    rda3m_right = nwbfile.devices["rDA3m (rAAV) (right NAcc)"]
+    assert isinstance(rda3m_right, Indicator)
+    assert rda3m_right.injection_coordinates_in_mm == (1.7, 1.7, -6.2)
+    assert rda3m_right.injection_location == "NAcc"
+    assert rda3m_right.label == "rAAV-hsyn-rDA3m"
+    assert rda3m_right.manufacturer == "BrainVTA"
+    assert rda3m_right.description.startswith("Recombinant AAV expressing the red-shifted dopamine sensor GRAB rDA3m")
+
+    # Check fiber photometry table
+    assert "fiber_photometry" in nwbfile.lab_meta_data
+    fiber_photometry_meta = nwbfile.lab_meta_data["fiber_photometry"]
+    assert isinstance(fiber_photometry_meta, FiberPhotometry)
+    assert isinstance(fiber_photometry_meta.fiber_photometry_table, FiberPhotometryTable)
+
+    table = fiber_photometry_meta.fiber_photometry_table
+    assert table.name == "fiber_photometry_table"
+    assert table.description == "fiber photometry table"
+
+    # Only recorded indicators are added to the table. We recorded from the fiber in left NAcc.
+    # 1. GACh4h (left NAcc) + Doric Purple LED
+    # 2. GACh4h (left NAcc) + Doric Blue LED
+    # 3. rDA3m (rAAV) (left NAcc) + Doric Green LED
+    expected_combinations = {
+        (gach4h_left, purple_led),
+        (gach4h_left, blue_led),
+        (rda3m_left, green_led)
+    }
+    combinations_in_table = set()
+
+    # Our table should have 3 rows, one for each indicator + excitation source combination
+    assert len(table) == 3
+
+    for row in table:
+        # The recording fiber, photodetector, and dichroic mirror are the same for all rows
+        assert row["optical_fiber"].item() == optic_fiber_left
+        assert row["photodetector"].item() == photodetector
+        assert row["dichroic_mirror"].item() == dichroic_mirror
+
+        # Each of the indicator + excitation source combinations should appear once
+        combinations_in_table.add((row["indicator"].item(), row["excitation_source"].item()))
+
+    assert expected_combinations == combinations_in_table
+
+
+def test_photometry_series_mappings():
+    # TODO: Test that each photometry series 
```

Comment:
This TODO remains. Punt to a separate issue & PR?

Reply:
either way sounds good. if it's quick and easy, might as well

Reply:
ok I changed my mind I will do that in a future PR lol